src/components/textarea/textarea.component.tsx

```diff
@@ -84,7 +80,7 @@ export interface TextareaProps
   label?: string;
   /** Inline label alignment */
   labelAlign?: "left" | "right";
-  /** Text applied to label help tooltip */
+  /** [Legacy] Text applied to label help tooltip */
```

Comment:
suggestion: might be worth adding that if you have the redesign flag set and no inputHint prop this will be used etc